### ISSUE TITLE
Add ons not showing in game at all

### ISSUE THREAD

**Issue by Baino123** [gw2-addon-loader/GW2-Addon-Manager#140]
### Please attach a copy of config.yaml or config.ini to the issue.

**Add ons not loading in game at all, menu keys for add ons also not working**

Have uninstalled/reinstalled add on mangers etc not working

**Comment by Pyrobandit19** [comment]
I have encountered the same issue, clean game and manager reinstall and still not working. Is you manager itself at least booting? Mine had github access error, then won't show at all.

**Comment by Northstars0422** [comment]
> I have encountered the same issue, clean game and manager reinstall and still not working. Is you manager itself at least booting? Mine had github access error, then won't show at all.

Mine also was giving me a github access error, and after a few times of it doing so now nothing happens when I attempt to open the launcher. Before I was getting the error, I was attempting to install the addons I wanted and it was closing the manager during the installation process without actually completing the installation.

**Comment by safrant** [comment]
I am seeing dsd9.dll installed into the bin64 directory and d3d11.dll in main GW2 directory, even though I have d3d11 enabled.  I then get the following in the gw2al_log.txt  

[INFO] core | Addon loader v1.0 (v1.1 r33) initialized
[INFO] core | Loading addons from "./addons" path
[INFO] core | Loading arcdps
[ERR] core | addons/arcdps/gw2addon_arcdps.dll not found
[INFO] core | Done loading addons
[WARN] core | Init called twice without proper unload. If your addon is not working, make sure you handle this situation!
[INFO] core | Unloading core addon
[INFO] core | Unloaded

**Comment by Aeron216** [comment]
Can confirm the same issue.

**Comment by Friendly0Fire** [comment]
The game always loads both versions, but addons should be able to handle this. ArcDPS however *does not currently work*.

**Comment by HeasHeartfire** [comment]
This is what the log says when I try to load GW2Radial. Addon manager v1.4.1.

Tried in both dx11 and dx9,

> [INFO] core | Addon loader v1.0 (v1.1 r33) initialized
> [INFO] core | Loading addons from "./addons" path
> [INFO] core | Loading d3d9_wrapper
> [INFO] core | Loaded d3d9_wrapper v2.0 r21
> [INFO] core | Loading gw2radial
> [WARN] core | Addon "gw2radial" is not loaded due to wrong dependency version "loader_core". Require v0.1(r1) but have v1.0(r33)
> [ERR] core | addons/gw2radial/gw2addon_gw2radial.dll dependency outdated
> [INFO] core | Done loading addons
> [INFO] d3d9_wrapper | Loading system dxgi
> [WARN] core | Init called twice without proper unload. If your addon is not working, make sure you handle this situation!
> [INFO] d3d9_wrapper | Loading system d3d11
> [INFO] core | Unloading core addon
> [INFO] core | Unloaded

**Comment by lcharrie** [comment]
You need to manually install GWRadial latest pre-release. The release pulled by the manager doesn't work with loader-core 1.1
https://github.com/Friendly0Fire/GW2Radial/releases/tag/v2.1.0-pre2

**Comment by HeasHeartfire** [comment]
Yeah you are right, my bad. I thought the pre-release was in the manager.

The newest release version is already present in the manager now so no need to do it manually.

**Comment by xivor** [comment]
Remember, addons are not necessarily ported over to DX11.  So, if using DX11 beta, and adding mods, you may not get any.  I can confirm that as of last night, ArcDPS works in DX9 and DX11.  I have confirmed that GW2Radial works in DX9, but not DX11.  I have not validated any others just yet, as I am trying to install and validate one software piece at a time. The most important thing is to make sure you install the latest prerelease listed above.

**Comment by Friendly0Fire** [comment]
As of earlier today GW2Radial is officially released so it is now available in Addon Manager as @HeasHeartfire mentioned :)

**Comment by xivor** [comment]
Thanks @Friendly0Fire I totally skimmed right over that!  Good to know!

**Comment by kamssiopeia** [comment]
Hey, i have exactly the same issue, using version 1.4.1 (1.4.2 doesn't work for me). I downloaded ArcDPS, d912pxy and GW2 radial nothing works on DX9 and Arc works on DX11. I reinstalled the whole game and it didn't help either

**Comment by SeiferLeonheart** [comment]
> 
> 
> Hey, i have exactly the same issue, using version 1.4.1 (1.4.2 doesn't work for me). I downloaded ArcDPS, d912pxy and GW2 radial nothing works on DX9 and Arc works on DX11. I reinstalled the whole game and it didn't help either
> 
> gw2al_log.txt
> 
> ```
> [INFO] core | Addon loader v1.0 (v1.1 r33) initialized
> [INFO] core | Loading addons from "./addons" path
> [INFO] core | Loading arcdps
> [INFO] core | Loaded d3d9_wrapper v2.0 r21
> [INFO] core | Loaded arcdps v1.0 r403
> [INFO] core | Loading d3d9_wrapper
> [INFO] core | Loading d912pxy
> [INFO] core | Loaded d912pxy v2.4 r703
> [INFO] core | Loading gw2radial
> [INFO] core | Loaded gw2radial v2.1 r1
> [INFO] core | Done loading addons
> [INFO] d3d9_wrapper | Loading system dxgi
> [WARN] core | Init called twice without proper unload. If your addon is not working, make sure you handle this situation!
> [INFO] d3d9_wrapper | Loading system d3d11
> [WARN] core | Init called twice without proper unload. If your addon is not working, make sure you handle this situation!
> [INFO] d3d9_wrapper | Loading system dxgi
> [WARN] core | Init called twice without proper unload. If your addon is not working, make sure you handle this situation!
> [INFO] d3d9_wrapper | Loading system d3d11
> [INFO] core | Unloading core addon
> [INFO] core | Unloaded
> ```
> 
> Edit: I tried 1.4.3 version and it didn't fix it

Exactly same log info here, line by line. Also tried 1.4.3.

**Comment by SleepyPwasnotavailable** [comment]
Same for me, nothing worked, updated to 1.4.3 version, then arc suddenly worked but nothing else (radial for example) and now nothing works again T-T

**Comment by blackhawk1542** [comment]
Hi, for me only arc is loading. Not sure why the rest doesn't load (clears, boon table, killproof.me, etc)

`[INFO] core | Addon loader v1.0 (v1.1 r33) initialized
[INFO] core | Loading addons from "./addons" path
[INFO] core | Loading arcdps
[INFO] core | Loaded d3d9_wrapper v2.0 r21
[INFO] core | Loaded arcdps v1.0 r402
[INFO] core | Loading d3d9_wrapper
[INFO] core | Done loading addons
[INFO] d3d9_wrapper | Loading system d3d9
[INFO] core | Unloading core addon
[INFO] core | Unloaded`

**Comment by blackhawk1542** [comment]
> Hi, for me only arc is loading. Not sure why the rest doesn't load (clears, boon table, killproof.me, etc)
> 
> [INFO] core | Addon loader v1.0 (v1.1 r33) initialized [INFO] core | Loading addons from "./addons" path [INFO] core | Loading arcdps [INFO] core | Loaded d3d9_wrapper v2.0 r21 [INFO] core | Loaded arcdps v1.0 r402 [INFO] core | Loading d3d9_wrapper [INFO] core | Done loading addons [INFO] d3d9_wrapper | Loading system d3d9 [INFO] core | Unloading core addon [INFO] core | Unloaded
> 
> I'm also using the 1.4.3 version, but none of the add-ons but arcDPS have worked since 1.4.1

Well today I uninstalled every add-on, deleted bin64 folder, deleted add-on manager folder, and installed everything again.
Somehow worked xD.

**Comment by Ejjitsu** [comment]
which bin64 

> > Hi, for me only arc is loading. Not sure why the rest doesn't load (clears, boon table, killproof.me, etc)
> > [INFO] core | Addon loader v1.0 (v1.1 r33) initialized [INFO] core | Loading addons from "./addons" path [INFO] core | Loading arcdps [INFO] core | Loaded d3d9_wrapper v2.0 r21 [INFO] core | Loaded arcdps v1.0 r402 [INFO] core | Loading d3d9_wrapper [INFO] core | Done loading addons [INFO] d3d9_wrapper | Loading system d3d9 [INFO] core | Unloading core addon [INFO] core | Unloaded
> > I'm also using the 1.4.3 version, but none of the add-ons but arcDPS have worked since 1.4.1
> 
> Well today I uninstalled every add-on, deleted bin64 folder, deleted add-on manager folder, and installed everything again. Somehow worked xD.

which bin64 folder did you delete i want to try the same thing but afraid illl kill off my reshader
